api/machine_types.go

```diff
@@ -407,6 +407,12 @@ var MachinePresets map[string]*MachineGuest = map[string]*MachineGuest{
 	"performance-16x": {CPUKind: "performance", CPUs: 16, MemoryMB: 16 * MIN_MEMORY_MB_PER_CPU},
 }
 
+func DefaultMachineGuest() *MachineGuest {
```

Comment:
looks like this function is not used after all

Reply:
oh yup! it was made redundant during a rebase and I totally forgot about it